src/app/settings.py

```diff
@@ -252,11 +270,11 @@ def show_toolbar(request):
 # Circle CI API
 CIRCLE_CI_CLIENT_ID = os.environ.get('CIRCLE_CI_CLIENT_ID')
 
-boto3_client = boto3.client(
+boto3_client = boto3.client( # TODO Remove, look for references.
```

Comment:
Yeah we should ensure that all comments/TODOs are in cards or otherwise addressed or on somebody's list.